rostock.py

```diff
@@ -0,0 +1,93 @@
+from bs4 import BeautifulSoup
+import urllib.request
+import re
+from datetime import datetime
+from datetime import date
+
+from pyopenmensa.feed import BaseBuilder
+from utils import Parser
+
+datePattern = re.compile(r"\w+ ([0-9]{2}\.[0-9]{2}\.[0-9]{4})")
+pricePattern = re.compile(u"(\\d+),(\\d+) €")
+
+# returns datetime.date object
+def parseDate(str):
+  match = datePattern.match(str)
+  dateStr = match.group(1)
+  dt = datetime.strptime(dateStr, "%d.%m.%Y")
+  #return dt
+  return date(dt.year, dt.month, dt.day)
+
+#def parseNotesOfMeal(td) # name = ""; for span in td.find("span"): text = firstChild; name += text; notes = span.select("small > sup"); format ist [kürzel,kürzel]
+# returns integer; in eurocents
+def parsePrice(priceAsString):
+  match = pricePattern.match(priceAsString)
+  return int(match.group(1)) * 100 + int(match.group(2))
+
+def parsePlan(url, internalMensaId, today):
+  #print(internalMensaId)
+  canteen = BaseBuilder()
+  dom = BeautifulSoup(urllib.request.urlopen(url).read(), 'lxml')
```

Comment:
If you move this into the loop, you can get rid of the second call to BS below and only need to set the `url` for the next day.